src/amo/sagas/home.js

```diff
@@ -83,7 +83,7 @@ export function* fetchHomeAddons({
       page_size: String(LANDING_PAGE_THEME_COUNT),
     },
   };
-  const popularExtensionsParams: SearchParams = {
+  const popularAddonsParams: SearchParams = {
     api: state.api,
     filters: {
       addonType: ADDON_TYPE_EXTENSION,
```

Comment:
I think this is meant to be `getAddonTypeFilter(ADDON_TYPE_THEME)`, in order to return themes and not extensions.

Reply:
@bobsilverberg Sorry I'm confused where this appears? Where are we attempting to direct users to popular themes?